spec/nylas/event_spec.rb

```diff
@@ -0,0 +1,152 @@
+# frozen_string_literal: true
+
+describe Nylas::Event do
+  describe ".from_json" do
```

Comment:
Tiny nit: If it's an instance method, should this be `#from_json`?

Reply:
It's a class method as we are using it like `described_class.from_json`

Reply:
Oh whoops, my mistake. There was an instance double so I thought it was an instance variable. Not a great assumption!